components/rank/index.tsx

```diff
@@ -0,0 +1,171 @@
+import styled from 'styled-components';
+import byteSize from 'byte-size';
+
+import { ArrowDown, ArrowUp } from '@components/icons';
+
+import device from '@utils/media_queries';
+import type { Data } from '@utils/dataType';
+
+const RankWrapper = styled.div`
```

Comment:
Consider splitting large css blocks into a `*.style.tsx` file

Reply:
We have less than 70 lines of styles here. I can't see why it could be a problem.